enterprise-beans/4.0/_index.md

```diff
@@ -46,36 +51,36 @@ See the [Jakarta Enterprise Beans 4.0 Release Plan](https://github.com/eclipse-e
 
```

Comment:
You need to list a Compatible Implementation.  In the case of EJB, you should be referencing the RC2 of Glassfish (same as your Compatibility Request).

Reply:
Done